src/mscorlib/corefx/System/Globalization/CompareInfo.Unix.cs

```diff
@@ -241,6 +241,73 @@ private bool EndsWith(string source, string suffix, CompareOptions options)
 
             return Interop.GlobalizationInterop.EndsWith(_sortHandle, suffix, suffix.Length, source, source.Length, options);
         }
+        
+        [System.Security.SecuritySafeCritical]
```

Comment:
These annotations are not necessary in CoreCLR/CoreRT. We have stripped them from CoreRT - since this piece is shared with CoreRT, you can strip them too.